shared/constants/bridge.ts

```diff
@@ -20,3 +20,12 @@ export const BRIDGE_API_BASE_URL = process.env.BRIDGE_USE_DEV_APIS
   : BRIDGE_PROD_API_BASE_URL;
 
 export const BRIDGE_CLIENT_ID = 'extension';
+
+export const ETH_USDT_ADDRESS = '0xdac17f958d2ee523a2206206994597c13d831ec7';
+export const METABRIDGE_ETHEREUM_ADDRESS =
+  '0x0439e60F02a8900a951603950d8D4527f400C3f1';
+
+export enum FeeType {
+  METABRIDGE = 'metabridge',
+  REFUEL = 'refuel',
+}
```

Comment:
This has been defined in ui/pages/bridge/types.ts already (all the types from the api are in that module)

Reply:
Done in [94c9486](https://github.com/MetaMask/metamask-extension/pull/27262/commits/94c9486f98160774a1d55cd448f058ec3f03d965)